service-proxy-examples/service-provider/pom.xml

```diff
@@ -7,76 +7,72 @@
   <parent>
     <groupId>io.vertx</groupId>
     <artifactId>service-proxy-examples</artifactId>
-    <version>3.5.2</version>
+    <version>3.5.1.redhat-004</version>
   </parent>
 
   <artifactId>service-provider</artifactId>
 
+  <properties>
+    <vertx.skip>false</vertx.skip>
+  </properties>
+
   <dependencies>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-web</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-service-proxy</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-rx-java</artifactId>
-      <version>${project.version}</version>
```

Comment:
This looks like it would need to be updated to `vertx-rx-java2` as `rx-java` is deprecated :)

Reply:
Sure. I will fix that